web-ui/src/main/resources/catalog/components/search/searchmanager/partials/searchresults.html

```diff
@@ -32,7 +34,9 @@
           data-ng-click="onClick(md)">
         <div data-ng-if="options.mode == 'title'">
           <gn-md-type-widget metadata="md"></gn-md-type-widget>
-          {{md.resourceTitle}}
+          <a href="catalog.search#/metadata/{{md.uuid}}">
```

Comment:
We have to improve this, sometime this component is used for selection also  
![image](https://user-images.githubusercontent.com/1701393/119977872-368d0a80-bfb9-11eb-88dc-749a22ec1117.png)

Link and selection does not play well with this... Improving it.